app/jobs/send_patient_digest_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# SendPatientDigestJob: sends assessment reminder to patients
+class SendPatientDigestJob < ApplicationJob
+  queue_as :mailers
+
+  def perform(*_args)
+    jurisdiction_addressees = {}
+    Jurisdiction.find_each do |jur|
+      if jur.send_digest
+        jurisdiction_patients = jur.all_patients.recently_symptomatic
+        jurisdiction_addressees = User.where(jurisdiction_id: jur.id, role: %w[super_user public_health public_health_enroller])
+        jurisdiction_addressees.each do |user|
```

Comment:
Minor tweak, but instead of `each`, use `find_each` here (https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each). That way we only pull batches from the database at a time, saving memory and other nice things.